Assets/PatchKit/UnityEditorExtension/Editor/Api/Models/Main/Wrapper.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace PatchKit.Api.Models.Main
+{
+
+[Serializable]
+public struct Wrapper<T>
```

Comment:
Please rename to `ApiArrayWrapper` since `Wrapper` as itself is too generic.

Reply:
done